pkg/ip/ip_test.go

```diff
@@ -664,7 +664,7 @@ func (s *IPTestSuite) TestPartitionCIDR(c *C) {
 	expectedRight := []*net.IPNet{createIPNet("10.128.0.0", 9, int(ipv4BitLen)),
 		createIPNet("10.64.0.0", 10, int(ipv4BitLen)),
 		createIPNet("10.32.0.0", 11, int(ipv4BitLen)),
-		createIPNet("10.16.0.0", 12, int(ipv4BitLen)),
+		createIPNet("10.11.0.0", 24, int(ipv4BitLen)),
```

Comment:
This seems like an unrelated change. This is test code and this delta is changing one of the test cases. Can you drop this change?